examples/conversation_create_message.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+import sys, os
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+
+import messagebird
+from messagebird.conversation_message import MESSAGE_TYPE_TEXT
+
+# ACCESS_KEY = ''
+# CONVERSATION_ID = ''
+# CHANNEL_ID = ''
+# TEXT_MESSAGE = ''
+
+try:
```

Comment:
Add this to examples in order to make sure code is not executed when imported.
if __name__ == "__main__":